starknet-rpc-test/call.rs

```diff
@@ -0,0 +1,232 @@
+#![feature(assert_matches)]
+
+extern crate starknet_rpc_test;
+
+use std::assert_matches::assert_matches;
+
+use rstest::{fixture, rstest};
+use starknet_accounts::Account;
+use starknet_contract::ContractFactory;
+use starknet_core::types::contract::SierraClass;
+use starknet_core::types::{BlockId, BlockTag, FunctionCall, StarknetError};
+use starknet_core::utils::get_selector_from_name;
+use starknet_ff::FieldElement;
+use starknet_providers::{MaybeUnknownErrorCode, Provider, ProviderError, StarknetErrorWithMessage};
+use starknet_rpc_test::constants::{ARGENT_CONTRACT_ADDRESS, FEE_TOKEN_ADDRESS, SIGNER_PRIVATE};
+use starknet_rpc_test::utils::create_account;
+use starknet_rpc_test::{ExecutionStrategy, MadaraClient, Transaction};
+
+#[fixture]
```

Comment:
I think a previous PR moved this fixture in a fixture.rs file. You will be able to use it if you rebase

Reply:
fixed for all tests

Reply:
Bless you 🙏